File: sort/selectionSort/eg1.java
import java.util.Scanner;
class psp
{
public static void main(String gg[])
{
int[] arr=new int[10];
for(int i=0;i<10;i++)
{
Scanner sc=new Scanner(System.in);
System.out.print("Enter a number :");
arr[i]=sc.nextInt();
}
for(int e=0;e<9;e++)
{
int si=e;
for(int f=e+1;f<10;f++)
{
if(arr[f]<arr[si]) si=f;
}
int g=arr[e];
arr[e]=arr[si];
arr[si]=g;
}
for(int i=0;i<10;i++) System.out.println(arr[i]);
}
}

File: sort/bubbleSort/eg1.cpp
#include<iostream>
using namespace std;
int main()
{
int arr[10];
for(int i=0;i<10;i++)
{
cout<<"Enter a number :";
cin>>arr[i];
}
for(int m=9;m>=0;m--)
{
for(int e=0,f=1;e<m;e++,f++)
{
if(arr[f]<arr[e])
{
int g=arr[e];
arr[e]=arr[f];
arr[f]=g;
}
}
}
for(int i=0;i<10;i++) cout<<arr[i]<<endl;
return 0;
}